web/app/scripts/src/services/fields-config.es.js

```diff
@@ -12,8 +12,6 @@ const module = angular.module(moduleProperties.moduleName, [
 module.service(moduleProperties.componentName, [
     function () {
         this.templatesBase = 'templates/incuna-surveys/fields';
-        this.headerTemplateUrl = `${this.templatesBase}/header.html`;
-        this.fieldsetHeaderTemplateUrl = `${this.templatesBase}/fieldset-header.html`;
```

Comment:
The fieldset now has it's own wrapper template that renders the title and fields.